target/linux/mediatek/mt7622/base-files/etc/hotplug.d/ieee80211/11_fix_wifi_mac

```diff
@@ -0,0 +1,18 @@
+[ "$ACTION" == "add" ] || exit 0
+
+PHYNBR=${DEVPATH##*/phy}
+
+[ -n $PHYNBR ] || exit 0
+
+. /lib/functions.sh
+. /lib/functions/system.sh
+
+board=$(board_name)
+
+case "$board" in
+	ruijie,rg-ew3200gx-pro)
+		hw_mac_addr=$(mtd_get_mac_ascii product_info ethaddr)
+		[ "$PHYNBR" = "0" ] && macaddr_add $hw_mac_addr 3 > /sys${DEVPATH}/macaddress
+		[ "$PHYNBR" = "1" ] && macaddr_add $hw_mac_addr 2 > /sys${DEVPATH}/macaddress
```

Comment:
@blocktrron  and @adschm  Do we have no better way to assign MACs from the U-Boot env to wifi devices?

Reply:
Yes, by setting `ethernet0 = &gmac0;` as alias in the DTS, then U-Boot will patch fdt to pass it down to Linux.